src/data/reviews.js

```diff
@@ -78,7 +98,8 @@ const exportedMethods = {
         const reviewCollection = await reviews();
 
         const review = await this.getReviewById(reviewId);
-        
+        let username = review.username;
+
         if(!review){
```

Comment:
I think it would be better if you check if the review exists before you define username.